[2025-03-06T23:09:55.229+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-06T23:09:55.240+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.upload_to_gcs scheduled__2025-03-06T22:00:00+00:00 [queued]>
[2025-03-06T23:09:55.246+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.upload_to_gcs scheduled__2025-03-06T22:00:00+00:00 [queued]>
[2025-03-06T23:09:55.247+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-06T23:09:55.261+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): upload_to_gcs> on 2025-03-06 22:00:00+00:00
[2025-03-06T23:09:55.266+0000] {standard_task_runner.py:72} INFO - Started process 6277 to run task
[2025-03-06T23:09:55.269+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline', 'upload_to_gcs', 'scheduled__2025-03-06T22:00:00+00:00', '--job-id', '192', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmpr60fmc4h']
[2025-03-06T23:09:55.271+0000] {standard_task_runner.py:105} INFO - Job 192: Subtask upload_to_gcs
[2025-03-06T23:09:55.285+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-06T23:09:55.310+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.upload_to_gcs scheduled__2025-03-06T22:00:00+00:00 [running]> on host b5502d51568b
[2025-03-06T23:09:55.312+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-06T23:09:55.371+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='upload_to_gcs' AIRFLOW_CTX_EXECUTION_DATE='2025-03-06T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-06T22:00:00+00:00'
[2025-03-06T23:09:55.372+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-06T23:09:55.384+0000] {data_pipeline.py:51} INFO - Executing script: /opt/airflow/scripts/upload_to_gcs.py
[2025-03-06T23:09:56.436+0000] {data_pipeline.py:55} ERROR - Error executing /opt/airflow/scripts/upload_to_gcs.py: Command '['python', '/opt/airflow/scripts/upload_to_gcs.py']' returned non-zero exit status 1.
[2025-03-06T23:09:56.437+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-06T23:09:56.445+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-06T23:09:56.446+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=upload_to_gcs, run_id=scheduled__2025-03-06T22:00:00+00:00, execution_date=20250306T220000, start_date=20250306T230955, end_date=20250306T230956
[2025-03-06T23:09:56.476+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-06T23:09:56.488+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-06T23:09:56.495+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
